tpgtools/serialization.go

```diff
@@ -1737,6 +1737,13 @@ func convertDataprocWorkflowTemplateBetaClusterClusterConfigSoftwareConfigToHCL(
 	if r.ImageVersion != nil {
 		outputConfig += fmt.Sprintf("\timage_version = %#v\n", *r.ImageVersion)
 	}
+	if r.OptionalComponents != nil {
```

Comment:
Meta, but do we have a way to either untrack this file or make diffs not show up in GitHub? It being auto-generated within the MM repo causes extra noise

Reply:
Filed https://github.com/hashicorp/terraform-provider-google/issues/10170, I know @ndmckinley had some thoughts about improving our use of `go generate`.